src/screens/MainScreen/index.js

```diff
@@ -20,10 +21,25 @@ const MainScreen = () => {
     getCalls();
   }, [currentUser]);
 
+  const reset = () => {
+    resetFlake(currentUser.id);
+    alert("Done! We're glad you're still interested!");
+  };
```

Comment:
This should be an async function and you should await the resetFlake before displaying the alert

Reply:
done.